base/uk.ac.stfc.isis.ibex.opis/src/uk/ac/stfc/isis/ibex/opis/DescriptionsProvider.java

```diff
@@ -151,5 +158,52 @@ public String guessOpiName(String name) {
 	    // If all else fails return an empty string
 		return "";
 	}
+
+    /**
+     * Get a Map of lists, where each key in the map is a category and each value is a list of items in that category.
+     * @return the map
+     */
+    public Map<String, List<String>> getOpiCategories() {
```

Comment:
This map will not change so it should not be recreated every time this method is called